nautobot_chatops/integrations/grafana/models.py

```diff
@@ -3,9 +3,26 @@
 from django.core.exceptions import ValidationError
 from django.core.serializers.json import DjangoJSONEncoder
 from django.utils.translation import gettext_lazy as _
-from nautobot.extras.utils import extras_features
+from nautobot.circuits import models as circuit_models
 from nautobot.core.models.generics import PrimaryModel, OrganizationalModel
-from nautobot_chatops.integrations.grafana.helpers import VALID_MODELS
+from nautobot.dcim import models as dcim_models
+from nautobot.extras import models as extra_models
+from nautobot.extras.utils import extras_features
+from nautobot.ipam import models as ipam_models
+from nautobot.tenancy import models as tenancy_models
+from nautobot.virtualization import models as virtualization_models
```

Comment:
Seems a bit heavy to import these as such.

Reply:
I will put a ticket in the backlog.